src/layout/AppMenu.vue

```diff
@@ -40,7 +42,10 @@ const model = computed<MenuItem[]>(() => {
     },
     showPosters && {
       label: 'Screens',
-      items: [{ label: 'Posters', icon: 'pi pi-fw pi-image', to: '/poster/list' }],
+      items: [
+        showGewisPosters && { label: 'GEWIS Posters', icon: 'pi pi-fw pi-image', to: '/poster/gewis' },
+        showHubblePosters && { label: 'Hubble Posters', icon: 'pi pi-fw pi-image', to: '/poster/hubble' },
+      ],
```

Comment:
If possible, can you make the label conditional? Namely if there is only one PosterHandler enabled, you can remove "GEWIS" and "Hubble" from the label. This keeps the current menu structure practically the same, as you will probably never have both enabled at the same time.

Reply:
I did consider this, but wasnt sure if i was allowed to make that assumption, and there was a chance there might be more poster handlers that _are_ enabled together with different slide sets. Such as a possible vertical poster handler for Hubble, so then this approach works a bit better? But is also in the future as the core isnt ready for that, so we can look at it later.

Reply:
yes this was added, when both are enabled it looks like the screenshot above, when only one is enabled it looks like before!